aztk/spark/models/plugins/jupyter_lab/jupyter_lab.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This custom script only works on images where jupyter is pre-installed on the Docker image
```

Comment:
We should change all of the plugins to be self-contained. Not sure that should be done in this PR, but we will have to change it soon anyways.

Reply:
Agreed, although this is no longer true since I'm install jupyter lab from conda. I copied the file over so that comment was unnecessary.